### PR TITLE
Rephrase feedback messages

### PR DESCRIPTION
**Pull request by yanbingtao**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/ArchiveCommand.java [hunk 1]
```diff
@@ -18,18 +18,18 @@
...
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Archives the employee identified by the index number used in the displayed employee list.\n"
+            + ": Archives the employee identified by the index number used in the displayed Employee Directory.\n"
             + "Parameter: INDEX (must be a positive integer)\n"
             + "Example: " + COMMAND_WORD + " 1";
 
     public static final String MESSAGE_ARCHIVE_PERSON_SUCCESS = "Archived Employee: %1$s";
     public static final String MESSAGE_PERSON_ALREADY_ARCHIVED = "This employee has already been archived!"
-            + "\nOnly employees in active list can be archived."
+            + "\nOnly employees in active(unarchived) list can be archived."
```
Comment (ureshiiYing) [inline src/main/java/seedu/address/logic/commands/ArchiveCommand.java]: leave a space between active and (unarchived)

**Review by ureshiiYing — COMMENTED** [review]
Only one minor change.

**Review by Diwu-Yi — APPROVED** [review]
LGTM

**Review by Persdre — APPROVED** [review]
LGTM

**Pull request merged**